_posts/ja/posts/2021-09-29-release-0.21.2.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Bitcoin Core 0.21.2 リリース
+name: blog-release-0.21.2
+id: ja-blog-release-0.21.2
+lang: ja
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 1
```

Comment:
Maybe this is supposed to be 0
"only true if announcement. **English posts only**"

Reply:
Thanks. Fix at 451ad1a.